Framework/TestHelpers/inc/MantidTestHelpers/ParallelRunner.h

```diff
@@ -57,6 +57,9 @@ void ParallelRunner::run(Function &&f, Args &&... args) {
     Mantid::Parallel::Communicator comm;
     f(comm, std::forward<Args>(args)...);
   } else {
+    // 1. Run serial
+    f(Mantid::Parallel::Communicator{}, std::forward<Args>(args)...);
```

Comment:
Two reasons to add this:
- Running code that is suposed to run in parallel should also work serially.
- Some static property type map in `Kernel` is constructed in a non-thread-safe manner, leading to segfaults unless something was run without threads first. If we do not do this here, *every* unit test must ensure to run something serially first.